### PR TITLE
Separating generated logs, updating job - archiving artifacts.

### PR DESCRIPTION
**Pull request by Katka92**
After https://github.com/pmacik/openshiftio-performance-common/pull/1 is merged, this branch could be merged too. After merging the branch, update job on osioperf-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/che-start-workspace to be able to archive artifacts from new location. Changes were made, just use job builder to update.

### CODE REVIEW

File: che-start-workspace/run.sh [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
+mkdir $LOG_DIR/png
+
 export COMMON="common.git"
 #rm -vf *.log
 #rm -vf *.csv
 #rm -vf *.png
 #exit 0
 
- git clone https://github.com/pmacik/openshiftio-performance-common $COMMON
+ git clone -b separate_logs https://github.com/Katka92/openshiftio-performance-common.git $COMMON
```
Comment (ScrewTSW) [inline che-start-workspace/run.sh]: Toto prepisat tak aby to pullovalo z `mastu`?

Comment (Katka92) [reply che-start-workspace/run.sh]: No zatím to v masteru není. Tak to buď můžu přepsat na master, ale s mergem se musí počkat, než to projde a nebo to nechat tak a až se to zmergne do masteru, tak to pak upravit...

Comment (ScrewTSW) [reply che-start-workspace/run.sh]: alebo to upravit tak aby to pouzivalo `master` a mergne sa to? :D

**Comment by Katka92** [comment]
@ScrewTSW fixed

**Pull request merged**